platform/kube/inject/inject.go

```diff
@@ -50,6 +50,7 @@ const (
 	istioSidecarAnnotationSidecarKey   = "alpha.istio.io/sidecar"
 	istioSidecarAnnotationSidecarValue = "injected"
 	istioSidecarAnnotationVersionKey   = "alpha.istio.io/version"
+	istioCertVolumeName                = "istio-cert"
```

Comment:
I feel this is internal and doesn't need to be configurable. As long as the name of Volume and that of VolumeMount match we should be fine.